PrayerBook/OrisonPresenter.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  OrisonPresenter.swift
+//  PrayerBook
+//
+//  Created by Дмитрий Беседин on 20.08.2020.
+//  Copyright © 2020 none-org. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol OrisonViewDelegate: NSObjectProtocol {
```

Comment:
Rename to OrisonViewProtocol and reminders about delegate

Reply:
i don't understand the second part of this commment(reminders about delegate)

Reply:
ok, I understand

Reply:
Thanks